config/willis.keymap

```diff
@@ -0,0 +1,6 @@
+// Copyright 2021 Manna Harbour
```

Comment:
Please update the dates in any files that are new or modified.

Reply:
Updated, thank you.